vetiver/pin_read_write.py

```diff
@@ -1,7 +1,17 @@
 import warnings
-
 from .vetiver_model import VetiverModel
-from .write_fastapi import _choose_version
+
+
+class ModelCard(UserWarning):
+    def __init__(
+        self,
+        message="""
+        Model Cards provide a framework for transparent, responsible reporting.
+        Use the vetiver `.qmd` Quarto template as a place to start,
```

Comment:
Does Quarto need more introduction?

Reply:
Probably not in this message, but can a link to <https://quarto.org/> be added to, for example, the docstring for this function?